config/application.rb

```diff
@@ -55,5 +55,8 @@ class Application < Rails::Application
     config.lograge.custom_options = lambda do |event|
       event.payload[:custom_log_items]
     end
+
+    config.sendgrid_api_user = ENV['SMTP_USERNAME']
+    config.sendgrid_api_key = ENV['SMTP_PASSWORD']
```

Comment:
Isn't it easier to put these in the sendgrid initializer? Then they don't even need adding to the rails config hash.